src/client/server_proxy.rs

```diff
@@ -267,6 +267,7 @@ impl ServerConnection {
                         self.node.network().remove(endpoint.resource_id());
                     }
                 },
+                _ => {}
```

Comment:
In version 0.14 this is unreachable for clients, and the `NetEvent::Connected` becomes reachable (since the connect method works asynchronously) and must be treated consequently. I think the changes on this side are not trivial.

Reply:
Me too i am not sure how to correct this one.